src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,45 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void setSize(){
+        int size = numbers.size();
+        assertThat(size).isEqualTo(4);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

HashSet은 같은 값이 있을 경우 같은 값으로 인식해 크기는 3

Reply:
기본적인 부분에서 실수를 했네요.
다음에는 코드를 더 꼼꼼히 살펴보도록 하겠습니다.

Reply:
> HashSet은 같은 값이 있을 경우 같은 값으로 인식해 크기는 3

기본적인 부분에서 실수를 했네요.
다음에는 코드를 더 꼼꼼히 살펴보도록 하겠습니다.